hpaction/tests/test_merge_issues.py

```diff
@@ -14,33 +14,31 @@ def mkissue(value: str):
 def test_it_merges_issues():
     assert merge_issue_models(
         [
-            mkissue(ISSUE_CHOICES.HOME__DOORBELL_BROKEN),
+            mkissue(ISSUE_CHOICES.HOME__PAINTING),
             mkissue(ISSUE_CHOICES.HOME__NO_HEAT),
             mkissue(ISSUE_CHOICES.HOME__NO_GAS),
         ],
         [HEAT_AND_GAS_MERGER],
     ) == [
-        Issue(area="HOME", description="Doorbell Broken", value="HOME__DOORBELL_BROKEN"),
-        Issue(area="HOME", description="No Heat & No Gas"),
+        Issue(area="HOME", description="Apartment needs painting", value="HOME__PAINTING"),
+        Issue(area="HOME", description="No heat & No gas"),
```

Comment:
Hmmm, @toolness is our enforcement of issue names using sentence case going to create some weird gramatical inconsistencies with HP? Looking here at this example of merging issue names...

Reply:
Oh I don't think so, as long as you fix those tests like you seem to be doing, everything should be fine.